static/app/views/starfish/queries/useApplicationMetrics.tsx

```diff
@@ -11,35 +8,18 @@ export type ApplicationMetrics = {
 };
 
 export const useApplicationMetrics = (_referrer = 'application-metrics') => {
-  const pageFilters = usePageFilters();
-  const {startTime, endTime} = getDateFilters(pageFilters);
-  const dateFilters = getDateQueryFilter(startTime, endTime);
-
-  const query = getQuery(dateFilters);
   const eventView = getEventView();
 
   // TODO: Add referrer
   const {isLoading, data} = useSpansQuery<ApplicationMetrics[]>({
     eventView,
-    queryString: query,
     initialData: [],
-    enabled: Boolean(query),
+    enabled: Boolean(eventView),
```

Comment:
Doesn't this mean it will always be enabled because `getEventView` always returns something.

Reply:
Yeah, I guess I should just set it to `true` eh?

Reply:
Yup! U can just remove the enabled property entirely I think. If we're matching the api of `useQuery` the default for enabled is true.

Only time we need enabled is if we're waiting for a certain condition to be met before executive the query, such as when you have a chain of dependent queries.

Reply:
In the other hooks where we use `useSpansQuery` enabled should probably be set if all the required parameters are passed into the hook and have values.

Reply:
There are a few places where `enabled` indirectly depends on the presence of a `span`, I updated those conditions and removed the rest

Reply:
Perfect!